centos/centos-7.x/centos-7.6-hpc/install.sh

```diff
@@ -22,3 +22,5 @@ set -ex
 # optimizations
 ./hpc-tuning.sh
 
+# copy test file
+./copy_test_file.sh
```

Comment:
Do you expect this to be different for different distros? If not, invoking the common copy script should be good enough.